scipy/stats/unuran/unuran_wrapper.pyx.templ

```diff
@@ -0,0 +1,869 @@
+    dpdf : callable
+        Derivative of the PDF w.r.t the variate. Must have the same signature
+        as the PDF.
+    mode : float, optional
+        Mode of the distribution. It is optional and only required if
+        `use_mode` is ``True``.
+    center : float, optional
+        Center (``loc``) of the distribution. It is optional and only
+        required if `use_center` is ``True``.
+    params : tuple, optional
+        Parameters that the PDF and DPDF take. Default is an empty tuple.
+    domain : list or tuple of length 2, optional
+        The support of the distribution. Must not contain `nan` values.
+        Default is ``None``. When ``None``, the support is assumed to be
+        :math:`(-\infty, \infty)`.
+    c : float, optional
+        The transformation function. See Notes for the allowed values for
+        this parameter and their interpretation.
+    cpoints : int, optional
+        The number of construction points. Default is 30.
+    variant : str, optional
+        The variant to use. Available options are:
+
+          * 'ps': PS variant (Default)
+          * 'gw': GW variant
+          * 'ia': IA variant
+
+    use_dars : bool, optional
+        If True, "derandomized adaptive rejection sampling" (DARS) is used
+        in setup. See [1]_ for the details of the DARS algorithm. Default
+        is True.
+    max_sqhratio : float, optional
+        Set upper bound for the ratio (area below squeeze) / (area below hat).
+        It must be a number between 0 and 1. Default is 0.99.
+    max_intervals : int, optional
+        Set maximum number of intervals. Default is 100.
+    use_center : bool, optional
+        Use the center as construction point, if given. Default is True.
+    use_mode : bool, optional
+        Use the (exact!) mode as construction point, if given. Default is
+        True.
+    guide_factor : float, optional
+        Set factor for relative size of the guide table for indexed search.
+        It must be greater than or equal to 0. When set to 0, then sequential
+        search is used. Default is 2.
+    seed : int, BitGenerator, Generator, RandomState, SeedSequence, optional
+        Seed for the underlying uniform random number generation.
+
+    Notes
+    -----
+    The transformation of the PDF must be concave in order to
+    construct the hat function. Such a PDF is called T-concave. Currently
+    the following transformations are supported:
+
+    .. math::
+
+        c = 0.: T(x) &= \log(x)\\
+        c = -0.5: T(x) &= \frac{1}{\sqrt{x}} \text{ (Default)}
+
+    Three variants of this method are available:
+
+    * GW: squeezes between construction points.
+    * PS: squeezes proportional to hat function. (Default)
+    * IA: same as variant PS but uses a compositon method with
+      "immediate acceptance" in the region below the squeeze.
+
+    
+    References
+    ----------
+    .. [1] UNU.RAN reference manual, Section 5.3.16,
```

Comment:
TBD if we want to refer to the UNU.RAN manual

references mentioned there are [GWa92] [HWa95] [HLD04: Cha.4]

you already have HWa95, we should add GWa92

Reply:
> TBD if we want to refer to the UNU.RAN manual
> 
> references mentioned there are [GWa92] [HWa95] [HLD04: Cha.4]
> 
> you already have HWa95, we should add GWa92

Sure, that sounds good. I will add the reference in the next commit.

Reply:
Done both for DAU and TDR